RUNNING.md

```diff
@@ -0,0 +1,72 @@
+# Running
+
+## Table of Contents
+
+- [Running With Docker](#running-with-docker)
+- [Environment Variables](#environment-variables)
+- [Troubleshooting](#troubleshooting)
+
+## Running With Docker
+
+For convenience,  `docker-compose.yml` configuration files are supplied with this project.
+The compose file orchestrates the Publisher application and the EQ schema validation service.
```

Comment:
The `eq-author-api` and `eq-publisher` projects have their own docker compose configuration files.

Until we hoist the configuration to the root of the repo, the documentation should reflect this.